### ISSUE TITLE
실험

### ISSUE THREAD

**Issue by chatterboy** [chatterboy/sad#10]

**Comment by chatterboy** [comment]
## 실험

### 베이스 모델
- DNN and shape 784 -> 800 -> 10
- batch_size: 100
- num_epochs: 10000
- learning_rate: 0.001
- zero initializer
- no activation function
test_accuracy: 0.925

### sigmoid 함수 추가
- shape 784 -> 800 -> sigmoid -> 10
test accuracy: 0.981

###  sigmoid 함수 추가 2
- shape 784 -> 800 -> sigmoid -> 10 -> sigmoid
test accuracy: 0.977

### relu 함수 추가
- shape 784 -> 800 -> relu -> 10
test accuracy: 0.097*
*학습이 다음과 같이 나타남
```
    ...
    epoch: 6000 - loss: 0.001525065628811717 - accuracy: 1.0
    epoch: 6200 - loss: 0.0028876769356429577 - accuracy: 1.0
    epoch: 6400 - loss: 0.002193036023527384 - accuracy: 1.0
    epoch: 6600 - loss: nan - accuracy: 0.15000000596046448
    epoch: 6800 - loss: nan - accuracy: 0.07000000029802322
    ...
    epoch: 10000 - loss: nan - accuracy: 0.07000000029802322
    Test phase - loss: nan - accuracy: 0.09799999743700027
````

## 결론
- 가장 마지막 레이어는 activation function를 적용하지 않는 것이 좋다
- 가장 마지막 레이어를 제외하고는 activation function을 적용하는 것이 좋다

**Comment by chatterboy** [comment]
## 실험

### 베이스 모델
- DNN and shape 784 -> 800 -> relu -> 10
- batch_size: 100
- num_epochs: 10000
- learning_rate: 0.01
- zero_initializer
test_accuracy: 0.097*
*학습이 다음과 같이 진행됨
```
    ...
    epoch: 6000 - loss: 0.001525065628811717 - accuracy: 1.0
    epoch: 6200 - loss: 0.0028876769356429577 - accuracy: 1.0
    epoch: 6400 - loss: 0.002193036023527384 - accuracy: 1.0
    epoch: 6600 - loss: nan - accuracy: 0.15000000596046448
    epoch: 6800 - loss: nan - accuracy: 0.07000000029802322
    ...
    epoch: 10000 - loss: nan - accuracy: 0.07000000029802322
    Test phase - loss: nan - accuracy: 0.09799999743700027
```

**Comment by chatterboy** [comment]
## 실험



## 결론

**Comment by chatterboy** [comment]
### 학습률 변경 & relu 추가 & weghts에 xavier_initializer 추가

- shape 784 -> 800 -> **relu** -> 800 -> **relu** -> 10
- **xavier_initializer**
- **learning_rate: 0.001**

test_accuracy: *
*epochs: 3800에서 0.98을 기록 후 diverge

**Comment by chatterboy** [comment]
## 실험

### 베이스 모델

- DNN and shape 784 -> 800 -> sigmoid -> 800 -> sigmoid -> 10
- batch_size: 100
- num_epochs: 100
- learning_rate: 0.01
- zero_intializer

test_accuracy: 0.974

## 결론

**Comment by chatterboy** [comment]
## Standard Neural Net 실험

### 베이스 모델

- DNN and shape 784 -> 800 -> sigmoid -> 800 -> sigmoid -> 10
- batch_size: 512
- num_epochs: 10000
- learning_rate: 0.01
- zero_initializer

test_accuracy: 0.982

**Comment by chatterboy** [comment]
## Dropout NN 실험

### 베이스 모델

- DNN and shape 784 -> 1024 -> sigmoid -> 1024 -> sigmoid -> 1024 -> sigmoid -> 10
- batch_size: 512
- num_epochs: 10000
- learning_rate: 0.01
- zero_initializer
- adam optimizer

test_accuracy: diverge

### dropout 추가

## 결론
